dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl

```diff
@@ -239,8 +239,36 @@ RUN for iter in {1..10}; do \
     (exit $exit_code)
 {{- end }}
 
-USER {{ .user }}
+{{- if (and (eq .Variant "wolfi-complete") (contains .from "wolfi"))  }}
+USER root
+# Install required dependencies from wolfi repository
+RUN for iter in {1..10}; do \
+        apk update && \
+        apk add --no-interactive --no-progress --no-cache nodejs-18=18.20.4-r0 npm=10.8.3-r0 glib dbus-glib libatk-1.0 \
+        libatk-bridge-2.0 cups-libs libxcomposite libxdamage libxrandr libxkbcommon pango alsa-lib \
+        font-opensans fontconfig gtk icu-data-full libnss mesa font-noto-cjk font-noto-emoji && \
+        exit_code=0 && break || exit_code=$? && echo "apk error: retry $iter in 10s" && sleep 10; \
+    done; \
+    (exit $exit_code)
+
+# Setup synthetics env vars
+ENV ELASTIC_SYNTHETICS_CAPABLE=true
+ENV TZ=UTC
+ENV NPM_CONFIG_PREFIX={{ $beatHome }}/.npm
+ENV PATH="$NPM_CONFIG_PREFIX/bin:$PATH"
+
+RUN echo \
+    $NPM_CONFIG_PREFIX \
+    {{ $beatHome }}/.config \
+    {{ $beatHome }}/.synthetics \
+    {{ $beatHome }}/.npm \
+    {{ $beatHome }}/.cache \
+    | xargs -IDIR sh -c 'mkdir -m 0770 -p DIR && chown -R {{ .user }} DIR'
```

Comment:
If we can keep the permission in sync with the other PR - https://github.com/elastic/beats/pull/40689/files#r1750771988

Reply:
Sorry, I have not seen the original comment https://github.com/elastic/beats/pull/40689/files#r1750771988 and thought it was a typo which I "fixed" in https://github.com/elastic/beats/commit/b06f7cea66b37978bc985a04aa851b6ea6fc2519

I think I need more context on how switching permissions `0770` to `0775` reflects removing the agent from the root group. The user created for this container is the owner of the entire directory tree and the last `0` in the bitmask denies access to "other" users and groups. Why should it be "read and execute" instead?

Reply:
I think @emilioalvap would be the right person to ask in this case 🙂